api/api/utils/throttle.py

```diff
@@ -116,14 +140,14 @@ def get_cache_key(self, request, view):
         if not (token_info and token_info.valid):
             return None
 
-        if token_info.rate_limit_model != self.applies_to_rate_limit_model:
+        if token_info.rate_limit_model not in self.applies_to_rate_limit_model:
```

Comment:
`not in` works for both a single string and for a list of strings, but I could see it being too flaky (`"standard" in "nonstandard"` is `True`). If folks prefer, I can either make all of them lists or change this to check the type of `self.applies_to_rate_limit_model` and using either `!=` or `not in`. Making all of them sets/tuples/lists would probably keep the code simpler overall.

Reply:
Done in https://github.com/WordPress/openverse/pull/3486/commits/297d74b67ada1763f4ffddd5fad8a669056432a8